### PR TITLE
Readme updates to help navigate SQLAlchemy test suites

### PR DESCRIPTION
**Pull request by hab6**

### CODE REVIEW

File: README.testsuite.md [hunk 1]
```diff
@@ -1,74 +1,146 @@
 # Introduction
 
-The SQLAlchemy GitHub repository contains two test suites:
```
Comment (clach04) [inline README.testsuite.md:3]: I think this should remain a colon

File: README.testsuite.md [hunk 2]
```diff
@@ -1,74 +1,146 @@
...
+The SQLAlchemy GitHub repository contains two test suites.
 
- - **Dialect Compliance Suite**
-     - [Readme](https://github.com/sqlalchemy/sqlalchemy/blob/main/README.dialects.rst)
-     - [Tests](https://github.com/sqlalchemy/sqlalchemy/tree/main/lib/sqlalchemy/testing/suite)
+**Dialect Compliance Suite**
```
Comment (clach04) [inline README.testsuite.md:5]: these should probably be numbered

File: README.md [hunk 3]
```diff
@@ -214,22 +214,6 @@ Setup test config
...
-             elif against(config, "oracle"):
-                 default = "READ COMMITTED"
-                 levels.add("AUTOCOMMIT")
-    +        elif against(config, "ingres"):
-    +            default = "READ COMMITTED"
-    +            levels.add("AUTOCOMMIT")  # probably needed, not sure what this is though - assuming tests are not commiting and expecting autocommit semantics
-             else:
-                 raise NotImplementedError()
-
 Run (all) tests:
```
Comment (clach04) [inline README.md:217]: Presumably this is no longer needed as changes already present? Information missing from commit. If this is the case, please update the PR description (no need to change history, unless you want to).

File: README.testsuite.md [hunk 4]
```diff
@@ -1,74 +1,146 @@
...
-## Overview and Setup
-
-The following text is from the page [README.dialects.rst](https://github.com/sqlalchemy/sqlalchemy/blob/main/README.dialects.rst) of the SQLAlchemy GitHub repository.
-
-> SQLAlchemy includes a [dialect compliance suite](https://github.com/sqlalchemy/sqlalchemy/tree/main/lib/sqlalchemy/testing/suite) that is usable by third party libraries, in the source tree at lib/sqlalchemy/testing/suite. There's no need for a third party dialect to run through SQLAlchemy's full testing suite, as a large portion of these tests do not have dialect-sensitive functionality. The "dialect compliance suite" should be viewed as the primary target for new dialects.
-
-### Environment and Packages
-These steps have been tested using the following environment:
+### Example Environment and Packages
+The operations described in this document have been tested in various environments up to and including the following versions:
```
Comment (clach04) [inline README.testsuite.md:23]: this is going to introduce a doc update burden - I'm not sure if this is worth the time spent on paperwork. I'm not against this per-se but I'm not for it.

**Review by clach04 — CHANGES_REQUESTED** [review]
Looking good, minor changes requested

File: README.testsuite.md (hunk 4 above)
Comment (clach04) [reply README.testsuite.md:23]: 👍 Works for me 😸

File: README.testsuite.md (hunk 1 above)
Comment (hab6) [reply README.testsuite.md:3]: Done

File: README.testsuite.md (hunk 2 above)
Comment (hab6) [reply README.testsuite.md]: Done

File: README.md (hunk 3 above)
Comment (hab6) [reply README.md]: Re-tested and confirmed that these changes are still required when using the Ingres connector with the unit tests of SQLAlchemy 1.x. The changes are not required with the 1.x dialect compliance suite nor with any of the SQLAlchemy 2.x tests (unit or dialect compliance).

Moved the instructions from README.md to README.testsuite.md.

**Comment by hab6** [comment]
@clach04 Thanks for your helpful comments so far. Could you please re-review when convenient? (not urgent)

**Review by clach04 — APPROVED** [review]

**Pull request merged**